### PR TITLE
Separate source file into ui.ts & main.ts

### PR DESCRIPTION
**Pull request by wtrgjadmw**
This PR separate the content of `index.ts` into `ui.ts` and `main.ts` for readability.

- `index.ts` only has `window.addEventListener('DOMContentLoaded', ...)` function and initializes events for html elements.
- `ui.ts` contains the functions related to HTML element, such as `document.getElementById` and `addEventListener`.
- `main.ts` contains the functions related to serial/USB port API, graph drawing, histogram painting and file downloading.

### CODE REVIEW

**Comment by wtrgjadmw** [comment]
moved unnamed functions in `index.ts` to `ui.ts` and added `window.addEventListener('DOMContentLoaded', ...)`!

**Comment by hikalium** [comment]
This change has some conflicts now so please rebase the changes and upload again!
Converting this p-r to draft for now (feel free to bring it back when it becomes ready!)

**Comment by wtrgjadmw** [comment]
fixed conflicts!

File: docs/servo_power_monitor/src/index.ts [hunk 1]
```diff
@@ -2,641 +2,60 @@
...
+  haltAddClickEvent,
+  requestSerialAddClickEvent,
+  requestUSBAddClickEvent,
+  selectDUTSerialAddClickEvent,
+  setPopupCloseButton,
+} from './ui';
+
+window.addEventListener('DOMContentLoaded', () => {
+  setPopupCloseButton();
+
```
[the remark this answers is not in the document]
Comment (wtrgjadmw) [reply docs/servo_power_monitor/src/index.ts]: remove them!

**Review by hikalium — APPROVED** [review]
LGTM. nice work!

**Pull request merged**